examples/custom-tea-opts/main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/charmbracelet/huh"
+)
+
+func main() {
+	var name string
+	if err := huh.NewForm(
+		huh.NewGroup(
+			huh.NewInput().Description("What is your name?").Value(&name),
+		),
+	).Run(); err != nil {
```

Comment:
This example is missing the actual options, right?

Reply:
hah, lol, yeah, was testing different ones and commited without any, fixing